### ISSUE TITLE
createColony.send is throwing Error: too many input arguments (count=6, expectedCount=1)

### ISSUE THREAD

**Issue by austinfrey** [JoinColony/colonyJS#467]
## Description
When trying to run the [getting started script](https://colony.io/dev/docs/colonyjs/intro-get-started#step-4-create-colony), calling `networkClient.createColony.send({tokenAddress})` is throwing an error for too many input arguments.
### the script
```colony.js
const {getNetworkClient} = require('@colony/colony-js-client')
const {open} = require('@colony/purser-software')
const {BN} = require('web3-utils')

;(async () => {
    const wallet = await open({
        privateKey: process.env.PRIVATE_KEY
    })

    const networkClient = await getNetworkClient('goerli', wallet)

    const createTokenTransaction = await networkClient.createToken.send({
        name: 'somename',
        symbol: 'TEST',
        decimals: 18
    })

    const tokenAddress = createTokenTransaction.meta.receipt.contractAddress
    
    const createColonyResponse = await networkClient.createColony.send({
        tokenAddress,
    })
})()
    .then(() => process.exit())
    .catch(err => console.log(err))
```
### the error
```bash
{ Error: too many input arguments (count=6, expectedCount=1)
    at Object.throwError (/home/austin/dev/punkmonk/colony-test/node_modules/ethers/utils/errors.js:74:17)
    at Interface.func (/home/austin/dev/punkmonk/colony-test/node_modules/ethers/contracts/interface.js:230:36)
    at /home/austin/dev/punkmonk/colony-test/node_modules/ethers/contracts/contract.js:88:31
    at EthersContract._callee3$ (/home/austin/dev/punkmonk/colony-test/node_modules/@colony/colony-js-adapter-ethers/lib/EthersContract.js:155:63)
    at tryCatch (/home/austin/dev/punkmonk/colony-test/node_modules/regenerator-runtime/runtime.js:62:40)
    at Generator.invoke [as _invoke] (/home/austin/dev/punkmonk/colony-test/node_modules/regenerator-runtime/runtime.js:296:22)
    at Generator.prototype.(anonymous function) [as next] (/home/austin/dev/punkmonk/colony-test/node_modules/regenerator-runtime/runtime.js:114:21)
    at step (/home/austin/dev/punkmonk/colony-test/node_modules/babel-runtime/helpers/asyncToGenerator.js:17:30)
    at /home/austin/dev/punkmonk/colony-test/node_modules/babel-runtime/helpers/asyncToGenerator.js:35:14
    at new Promise (<anonymous>)
  reason: 'too many input arguments',
  code: 'UNEXPECTED_ARGUMENT',
  count: 6,
  expectedCount: 1 }
```

- [x] TODO write description

**Comment by austinfrey** [comment]
This also occurs when trying to run the [colony-starter](https://github.com/JoinColony/colonyStarter/tree/develop/packages/colony-starter) boilerplate

**Comment by austinfrey** [comment]
Just dropping a note that it's this line:  https://github.com/JoinColony/colonyJS/blob/v1.14.6/packages/colony-js-client/src/ColonyNetworkClient/index.js#L1118 introduced in this commit: 0588255b753cca1c4b4b83b57d96761e6f7eaf31

Having the values other than `tokenAddress` present seems to be causing this error in the underlying ether.js module.

**Comment by chmanie** [comment]
Hey! I have just released `@colony/colony-js@2.0.0` which should not have this issue. Check out the docs here:

https://joincolony.github.io/colonyJS/

Please do not use the colonyStarter anymore, we are deprecating this project.
